modules/katello_content_view.py

```diff
@@ -81,22 +96,40 @@
     repositories:
       - name: 'Fedora 26'
         product: 'Fedora'
+
+- name: "Create a composite content view
+  katello_content_view:
+    username: "admin"
+    password: "changeme"
+    server_url: "https://foreman.example.com"
+    name: "Fedora CCV"
+    organization: "My Cool new Organization"
+    composite: true
+    auto_publish: true
+    components:
+      - name: Fedora CV
+        version: 1.0
+      - name: Internal CV
+        latest: true
```

Comment:
The example code shows components defined as a dictionary with properties "name" and "version", however in line 214 the code expects a "content_view" property